### DISCUSSION TITLE
direct DDL is disabled trying to push on branch that does not have safe migration enabled

### DISCUSSION THREAD

**Discussion by etdej** [planetscale/discussion#618] [Bug Reports] [answered]
Hi,

Nothing changed to my CICD pipeline and I started to have "direct DDL is disabled" this morning when trying to push on non main branch without sale migration enabled.

Any known active bug on your side ?

Thanks,

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @etdej, no this is not a known active bug on our side.

Could you open a support request with more details? We'll look into it. https://support.planetscale.com/hc/en-us/requests/new

Thank you
